src/index.js

```diff
@@ -1,16 +1,22 @@
 import {dispatch} from './lib/utils.js'
 import ConfirmationController from './lib/ConfirmationController.js'
 
-const confirm = (_message, _formElement, submitter) => {
-  const confirmationResponse = dispatch('confirm', submitter)
+const confirm = async (message, _formElement, submitter) => {
+  if (!window._TurboConfirm) init()
+
+  const confirmationResponse = await window._TurboConfirm.perform(message, submitter)
+
+  if (confirmationResponse) {
+    dispatch('confirm-accept', submitter)
+  } else {
+    dispatch('confirm-reject', submitter)
+  }
 
-  dispatch('after-confirm', submitter, {detail: confirmationResponse})
   return confirmationResponse
 }
 
 const init = (options) => {
-  const controller = new ConfirmationController(options)
-  document.addEventListener('rms:confirm', (event) => controller.perform(event))
+  window._TurboConfirm = new ConfirmationController(options)
```

Comment:
is enforcing a singleton required?

I could see wanting one configuration to be used by default for any streams, and another one used for things I initiate manually

Reply:
@seanpk could you elaborate on wanting multiple configurations?  I'm struggling to see the use-case for that.  Thanks!

Reply:
if you were manually going to invoke a confirm dialog, you might want to have a different configuration than what is set to be used for `data-turbo-confirm` with `Turbo.setConfirmMethod()`

(we have a design where clicking a button reveals a confirm dialog in a tooltip; it is desirable to leverage `@rolemodel/turbo-confirm` for that case, and it also is/seems desirable to have independent configuration for the "custom" case and the "regular" case)

Reply:
@seanpk okay, that makes sense. Thanks for explaining.

Any thoughts on how that should work?  Would you expect to instantiate a new object w/ config for manual use in like a stimulus controller, or something like that?  

I can't see making that work without also changing the regular Turbo setup slightly, but I think that's fine.  I expected this would become v2.0.